lib/foreman_maintain/repository_manager.rb

```diff
@@ -0,0 +1,13 @@
+require 'foreman_maintain/repository_manager/el'
+include ForemanMaintain::Concerns::OsFacts
+module ForemanMaintain
```

Comment:
the `include` looks weird outside the `module`, is that intentional?

Reply:
I could be wrong, but the concern OsFacts is module and we are calling that in another module. If I include that in ForemanMaintain module then methods are not available.

Reply:
Not saying this is wrong (I don't actually know), just that it looked weird to me.